apel/db/records/storage.py

```diff
@@ -88,3 +91,121 @@ def get_db_tuple(self, source=None):
         The source (DN of the sender) isn't used in this record type currently.
         """
         return Record.get_db_tuple(self)
+
+    def get_ur(self, withhold_dns=False):
```

Comment:
could this be done with a `for` loop? See this similar method for [DataSetSummaryRecord.get_ur](https://github.com/gregcorbett/apel/blob/dataset_accounting_v2/apel/db/records/dataset_summary.py#L76)

Reply:
Probably. This was a copy-paste job from the manual changes that had been made on the old pre-prod host that I knew worked so didn't redo it.

Reply:
okay